### PR TITLE
Fix event breadcrumbs

### PR DESCRIPTION
**Pull request by mpelzsherman**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/15880

## Testing done
unit

## Screenshots
Screen Shot 2020-12-08 at 12 48 48 PM

## Acceptance criteria
- [x] Breadcrumbs on Events pages in VAMC and Outreach Hub should reflect the H1 title (no hyphens, unless inherent in the name) 

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by mpelzsherman** [comment]
The event pages have an extra breadcrumb at the end which we are replacing here with the page title text.

It would be nice to know why the event data is different from other CMS bundle types.

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -441,16 +441,24 @@ module.exports = function registerFilters() {
...
 
+  // TODO - add unit tests for this!
   liquid.filters.deriveLastBreadcrumbFromPath = (
     breadcrumbs,
     string,
     currentPath,
+    replaceLastItem = false,
   ) => {
     const last = {
       url: { path: currentPath, routed: true },
```
Comment (zacharymorel) [inline src/site/filters/liquid.js:452]: What's the `routed: true` property for?

**Review by zacharymorel — APPROVED** [review]
LGTM, Unit test passes.

File: src/site/filters/liquid.js (hunk 1 above)
Comment (mpelzsherman) [reply src/site/filters/liquid.js:452]: Not sure, but we didn't change it in the PR.

Comment (ethanteague) [reply src/site/filters/liquid.js:452]: It's a cms thing. It means Drupal has created a more human readable path alias for the page.

File: src/site/filters/liquid.js [hunk 2]
```diff
@@ -441,16 +441,24 @@ module.exports = function registerFilters() {
     return null;
   };
 
+  // TODO - add unit tests for this!
```
Comment (U-DON) [inline src/site/filters/liquid.js:444]: Looks like a test was added. Does this still apply?

**Review by U-DON — APPROVED** [review]

File: src/site/filters/liquid.js (hunk 2 above)
Comment (brangi) [reply src/site/filters/liquid.js:444]: not anymore, will remove

File: src/site/filters/liquid.unit.spec.js [hunk 3]
```diff
@@ -73,3 +73,28 @@ describe('dateFromUnix', () => {
...
+        origBreadCrumbsList,
+        title,
+        'http://va.gov',
+        true,
+      )
+      .pop();
+
+    expect(last.text).to.eq(title);
+  });
+});
```
Comment (mpelzsherman) [inline src/site/filters/liquid.unit.spec.js:100]: Can we add another test where `replaceLastItem` is `false`?

Comment (mpelzsherman) [reply src/site/filters/liquid.unit.spec.js:100]: Also let's add some tests where `origBreadCrumbsList` has fewer or greater than 3 elements.

**Review by brangi — APPROVED** [review]

File: src/site/filters/liquid.unit.spec.js [hunk 4]
```diff
@@ -73,3 +73,85 @@ describe('dateFromUnix', () => {
...
+        origBreadCrumbsList3,
+        title,
+        'http://va.gov',
+        true,
+      )
+      .pop();
+
+    expect(last1.text).to.eq(title);
+    expect(last2.text).to.eq(title);
+    expect(last3.text).to.eq(title);
```
Comment (mpelzsherman) [inline src/site/filters/liquid.unit.spec.js:136]: These should probably be broken out as separate `it` blocks.

**Pull request merged**